src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <article className={classNames(PostInfo)}>
```

Comment:
```suggestion
    <article className="PostInfo">
```

Fix it pls, what for you need classNames library here?